h/static/scripts/test/time-test.js

```diff
@@ -50,6 +51,34 @@ describe('time', function () {
   });
 
   describe('.toFuzzyString', function () {
+
+    var originalIntl;
+
+    before(function () {
+
+      // Intl isn't implemented in PhantomJS so we have to mock it in these
+      // tests. The mock version returns the results we'd expect from the real
+      // one for the test values that we use.
+      originalIntl = window.Intl;
```

Comment:
@seanh - In general, there are a few approaches you can take for passing dependencies down a chain of functions that I know of:

1. Just pass the dependency down explicitly as an argument to each function
2. Do the same as (1) but bundle the dependencies up into a context object (ie. a record/object which maps keys to dependencies). Pyramid's `request` object or Angular's `$scope` are effectively instances of this.
3. Instead of passing dependency 'A' to 'B' which calls 'C' that in turn uses 'A', you instead create a version of 'C' that has 'A' already resolved and pass that in to 'B'. ie. You would make the `format()` function itself an argument to `toFuzzyString()`. This is what you're effectively doing with Angular's dependency injection.